### PR TITLE
Added scroll to top button

### PR DESCRIPTION
**Pull request by pwill12**
I added a scroll to top button on the item list components...when a user create a search the button comes on..you can check it out..let me know if there's any errors..Also working on updating the react codebase.

### CODE REVIEW

**Comment by Sanix-Darker** [comment]
Hey there and thank you for your Pull Request !
Sorry for the late reply !
Is it possible to split these changes into multiple separated commits ?
Because am seeing formatting along your changes and some other updates !
And at the end a screenshot or an animated gif for the scroll to the top feature 🙏🏼 in the Pull Request description ! 

Thanks in advance !

**Comment by pwill12** [comment]
No..its not possible...i only made changes in the app.css files which i got rid of...
Annotation 2022-08-21 045137
Annotation 2022-08-21 045304
Annotation 2022-08-21 045348
Annotation 2022-08-21 045421
Annotation 2022-08-21 045553
Annotation 2022-08-21 045921

**Comment by Sanix-Darker** [comment]
Thanks for the screenshots !
Looks like there is some conflicts with the main branch, feel free to open a new Pull Request when you can or resolve those here 🙏🏼

**Review by pwill12 — COMMENTED** [review]
Resolved the conflicts...

File: src/App.css [hunk 1]
```diff
@@ -1,97 +1,121 @@
-body{
```
Comment (Sanix-Darker) [inline src/App.css:1]: this file should not belong to this PR ! 🙏🏼

File: src/components/ItemList/index.js [hunk 2]
```diff
@@ -290,6 +290,27 @@ class ItemList extends Component {
...
+                    fontSize: "11px",
+                    bottom: "40px",
+                    right: "40px",
+                    backgroundColor: "grey",
+                    color: "#fff",
+                    textAlign: "center",
+                  }}
+                >
+                  Scroll to top
+                </button>
```
Comment (Sanix-Darker) [inline src/components/ItemList/index.js:313]: 👍🏼 This is amazing and should be the only changes coming from this Pull Request !

**Review by Sanix-Darker — CHANGES_REQUESTED** [review]
Just like the yarn.lock, some file should not belong in this pull request

File: src/components/ItemList/index.js (hunk 2 above)
Comment (pwill12) [reply src/components/ItemList/index.js:313]: Yeah thats the only changes coming from the pull request.

Comment (Sanix-Darker) [reply src/components/ItemList/index.js:313]: So, you should remove everything except for those lines, that's what I was requesting up there!
But in the pull request, we still have more than 7k line changes!
Feel free to open a new pull request containing only these 22 lines! 🙏🏽
